.github/workflows/api-swagger.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v4
       with:
-        go-version: '1.20'
+        go-version: '1.21'
```

Comment:
In the CI go1.21 is used, but the mod says 1.22. Is this expected?

Reply:
Nope, I overlooked that, it should be 1.22. I'll change it.